docs/claws-mail.md

```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
+Then we need to create some folders:
+
+    mkdir -p ~/.vdirsyncer/status
+    mkdir ~/.contacts
+
+## Configuration
+Now we create the configuration for the vdirsyncer:
+
+    vim ~/.vdirsyncer/config
+
+The config should look like this
+```ini
+[general]
+status_path = "~/.vdirsyncer/status/"
+
+[storage local]
+type = "singlefile"
+path = "~/.contacts/%s.vcf"
+
+[storage online]
+type = "carddav"
+url = "CARDDAV_LINK"
+username = "USERNAME"
+password = "PASSWORD"
+
+[pair contacts]
+a = "local"
+b = "online"
+collections = ["from a", "from b"]
+conflict_resolution = "b wins"
+```
+
+First of all in the general section, we define the status folder path, for some logging information from vdirsyncer. 
```

Comment:
It's technically not logging, but discovered collections and generally stuff that needs to persist between syncs.